src/box/txn_limbo.h

```diff
@@ -480,6 +480,12 @@ txn_limbo_unfence(struct txn_limbo *limbo);
 void
 txn_limbo_init();
 
+/**
+ * Denitialize qsync engine.
+ */
+void
+txn_limbo_destroy();
```

Comment:
Should be named `txn_limbo_free` according to our naming convention. Anyway, there are so many susbystems touched in this commit with so different cases so it may be better to split this commit as well and submit it in a different PR.